Functions/xASL_init_DefineStudyData.m

```diff
@@ -124,14 +124,20 @@
 % ------------------------------------------------------------------------------------------------
 %% 3) Create list of total baseline & follow-up subjects, before exclusions
 x.dataset.nSessions = length(x.SESSIONS);
-x.SUBJECTS = x.dataset.TotalSubjects; % temporarily for xASL_init_LongitudinalRegistration
+if ~isempty(x.dataset.TotalSubjects)
+    % Temporarily fix for xASL_init_LongitudinalRegistration
+    x.SUBJECTS = x.dataset.TotalSubjects;
+end
+if ~isfield(x,'SUBJECTS')

```

Comment:
These things are difficult; if there is no SUBJECTS field because something went wrong, it makes no sense to create an empty one here?
In fact, this should give a warning, and should be combined as an `elseif` in the previous if statement at line 127

Reply:
Add a warning below line 131 and make sure that the program does what it should afterwards.